### ISSUE TITLE
Use Message component in IE11 throws exceptions

### ISSUE THREAD

**Issue by reverland** [ElemeFE/element#266]
### ElementUI version
1.0.0-rc.5
### OS/Browers version
win 10/IE 11

### Vue version
2.0.0

### Reproduction Link

https://codepen.io/reverland/full/ORvLrv

### Steps to reproduce

Just see the reproduction link in IE11.

### What is Expected?

no error is throwed

### What is actually happening?

Throw error:
    Unable to get property 'toLowerCase' of undefined or null reference

I inspect this more, and looks that IE11 has no Function.prototype.name support? So, Message component will have undefined name property.

However, how `element.eleme.io` works well...

**Comment by Leopoldthecoder** [comment]
It appears your reproduction has no `Message` component in it. Please check again.

**Comment by reverland** [comment]
https://codepen.io/reverland/pen/ORvLrv

```html
<script src="//unpkg.com/vue@2.0.0/dist/vue.js"></script>
<script src="//unpkg.com/element-ui@1.0.0-rc.5/lib/index.js"></script>

<div id="app"></div>
```
It include `Message`
I think `Message` is where problem lies, so I title it like this.

**Comment by reverland** [comment]
https://github.com/ElemeFE/element/blob/master/src/index.js#L108 

`Message.name` is undefined in IE11(or below maybe) but not Edge/Firefox/Chrome

**Comment by Leopoldthecoder** [comment]
I see what you mean. Our mistake, `Message` should not be registered as a component.

**Comment by Leopoldthecoder** [comment]
fixed in #269
